koswat/core/io/csv/koswat_csv_multi_header_fom.py

```diff
@@ -0,0 +1,18 @@
+from koswat.core.io.csv.koswat_csv_fom_protocol import KoswatCsvFomProtocol
+
+
+class KoswatCsvMultiHeaderFom(KoswatCsvFomProtocol):
+    headers: list[list[str]]
+    entries: list[list[str]]
+
+    def __init__(self) -> None:
+        self.headers = []
+        self.entries = []
```

Comment:
If you are not going to use the `__init__` you might as well make this into a `dataclass` with default values (you might have noticed I'm doing it whenever I come upon such initializers).